### PR TITLE
added API key to dotenv, created basic nav bar and footer to attribute API use to TMDB

### PR DESCRIPTION
**Pull request by halimahexe**
As the title says, I got an API key from TMDB to pull movie data when a user searches (in form component for adding movies to the database).

I created a basic nav bar so the webpage shows something when `npm run dev` is run. And added attribution to a footer component.

### CODE REVIEW

File: .env.template
Comment (Feargh) [inline .env.template:1]: I'm going to update this with this code
```
API_KEY="TMDB API Key"
API_TOKEN="TMDB API Token"
```

**Review by Feargh — CHANGES_REQUESTED** [review]
.env.template needs the code added which I will do now.

File: .env.template
Comment (Feargh) [reply .env.template:1]: Code is now updated.

**Review by Feargh — APPROVED** [review]
All set up code looks good!
.env.template is interesting, I've never seen this before.

**Review by AlexPD93 — APPROVED** [review]

**Pull request merged**